tests/framework/installer/ceph_manifests.go

```diff
@@ -247,7 +248,10 @@ spec:
 }
 
 // GetRookOperator returns rook Operator manifest
-func (m *CephManifestsMaster) GetRookOperator(namespace string) string {
+func (m *CephManifestsMaster) GetRookOperator(namespace, enableCSI string) string {
```

Comment:
1) Should `enableCSI` still be in this param? It seems like the decision was made to always enable CSI.

2) This should be a `bool` parameter, not a string.

Reply:
Agreed that it should be a bool instead of a string.